Assets/Scripts/MainMenu.cs

```diff
@@ -5,8 +5,27 @@
 
 public class MainMenu : MonoBehaviour
 {
+    public enum GameScenes
```

Comment:
actually, let's make these private constants instead, that way you won't need to cast.

Reply:
so something like:

```
private const int MainScene = 1;
```

and then reference it by doing

`SceneManager.LoadScene(MainScene);`